lib/stringify.js

```diff
@@ -94,6 +92,21 @@ var stringify = function stringify(
         return [formatter(prefix) + '=' + formatter(String(obj))];
     }
 
+    if (generateArrayPrefix === 'comma' && isArray(obj)) {
+        if (formatter !== formats.formatters['RFC3986']) {
+            obj = obj.join(',');
+        } else {
+            if (encoder) {
+                var commaKey = encodeValuesOnly ? prefix : encoder(prefix, defaults.encoder, charset, 'key');
+                var commaValue = obj.map(function (item) {
```

Comment:
we can't add a dependency on `.map` here.

Reply:
what is wrong with the `.map`?

Reply:
@petersolopov Pre-ES5 browsers won't have it. You can rebase and use `utils.maybeMap` instead.